packages/tasit-contracts/README.md

```diff
@@ -0,0 +1,3 @@
+# `tasit-sdk`
+
+Most of the details are in the main README for the monorepo [here](https://github.com/tasitlabs/TasitSDK/blob/develop/README.md)
```

Comment:
Random comment - I could see the `tasit-contracts` package that has a collection of the latest contracts from major projects becoming a popular standalone utility that other developers might use for testing other libaries too.

Reply:
Yes, I agree.

Reply:
I created this issue to remind myself to mention that in the README https://github.com/tasitlabs/TasitSDK/issues/65